rgd-client/tests/data_fixtures.py

```diff
@@ -0,0 +1,28 @@
+import subprocess
+
+import pytest
+
+from . import MANAGE_PATH, PYTHON_PATH
+
+sources = [
+    'rgd_3d_demo',
+    'rgd_fmv_demo',
+    'rgd_fmv_wasabi',
+    'rgd_geometry_demo',
+    'rgd_imagery_demo',
+    'rgd_imagery_landsat_rgb_s3',
```

Comment:
Let's definitely not include `rgd_imagery_landsat_rgb_s3` or `rgd_fmv_wasabi` when testing as these are larger demo datasets not meant for tests

These two are likely the reason it is running so slow